kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/LlvmDeclarations.kt

```diff
@@ -343,7 +343,7 @@ private class DeclarationsGeneratorVisitor(override val generationState: NativeG
                 it.setLinkage(LLVMLinkage.LLVMCommonLinkage) // Allows to be replaced by other bitcode module.
             }
         } else {
-            staticData.createGlobal(writableTypeInfoType, "")
+            staticData.createGlobal(writableTypeInfoType, "ktypew:${declaration.fqNameForIrSerialization}")
```

Comment:
It makes sense to add an `#internal` suffix. Otherwise, it might clash with something from the exported case above.
LLVM handles such a clash, but if the exported name is encountered later than the clashing internal name, then it will get mangled.